interface/src/main/java/xsbti/Logger.java

```diff
@@ -5,9 +5,9 @@
 
 public interface Logger
 {
-	public void error(F0<String> msg);
-	public void warn(F0<String> msg);
-	public void info(F0<String> msg);
-	public void debug(F0<String> msg);
-	public void trace(F0<Throwable> exception);
+	void error(F0<String> msg);
```

Comment:
In a Java interface, methods are implicitly `public`, values are implicitly `public static final`, so public is not needed :)